### ISSUE TITLE
CHOOSE A THEME OR START FROM SCRATCH

### ISSUE THREAD

**Issue by jechnology** [jechnology/mms142-groupe-2020#2]
GitHub has an in-built website creator called GitHub Pages. To save time, I've screenshotted the themes and added them here. Please vote on which theme you like best. Then comment back here to vote your chosen theme. Whichever theme gets most votes would be the one we will use for the website. 

Update: I recently discovered how to host a website (made from HTML and CSS files) in GitHub, so we now make the option of starting our website from scratch, available. 

Votes:

Scratch (HTML and CSS) - 1 (@jechnology ) 

Cayman Theme -
Slate Theme -
Merlot Theme -
Time Machine Theme -
Minimal Theme - 1 (@juliaalexis )
Leap Day Theme -
Modernist Theme -
Hack Theme -
Midnight Theme -
Architect Theme -
Tactile Theme -
Dinky Theme -

![Themes](https://user-images.githubusercontent.com/74533082/100233422-b41d8280-2f64-11eb-9fa7-61b5be226651.png)

**Comment by gerolddelacruz** [comment]
Hey @jechnology I don't think we have access to the settings of this repository since it belongs to you. I'm okay with you deciding on the theme of our site.

**Comment by jechnology** [comment]
Oh is that so? Guess I have lots more to learn about Github 😅. Should I screenshot the themes and comment it here? I really don't want to decide by myself.

**Comment by juliaalexis** [comment]
I vote for the minimal theme!

**Comment by jechnology** [comment]
@juliaalexis may I ask how you've known about the theme? Can you access the Settings for this repository or is it just like what @gerolddelacruz said that only the owner has access to the settings?

**Comment by juliaalexis** [comment]
Heyo. I tinkered with GitHub the other day and tried to create my own repository which was how I learned about the themes available. Comparing my experiment project with this one, @gerolddelacruz is right. We can't access the settings portion :( I added a screenshot of the two below. 
![image](https://user-images.githubusercontent.com/74344419/100233970-63f2f000-2f65-11eb-8b97-98d7754ab0eb.png)
![image](https://user-images.githubusercontent.com/74344419/100233948-5c334b80-2f65-11eb-84bf-77c60e6eeb29.png)
